website/docs/r/databricks_access_connector.html.markdown

```diff
@@ -51,7 +51,13 @@ The following arguments are supported:
 
 An `identity` block supports the following:
 
-* `type` - (Required) The type of identity to use for this Access Connector. `SystemAssigned` is the only possible value.
+* `type` - (Required) Specifies the type of Managed Service Identity that should be configured on this Access Connector. Possible values are `SystemAssigned`, `UserAssigned`, `SystemAssigned, UserAssigned` (to enable both).
+
+* `identity_ids` - (Optional) Specifies a list of User Assigned Managed Identity IDs to be assigned to this Access Connector.
+
+~> **NOTE:** This is required when `type` is set to `UserAssigned` or `SystemAssigned, UserAssigned`.
```

Comment:
at this time the Service doesn't actually support `SystemAssigned, UserAssigned` even though the Swagger defines it (I've sent https://github.com/Azure/azure-rest-api-specs/pull/23370 to fix that) - so we can make this:

```suggestion
* `type` - (Required) Specifies the type of Managed Service Identity that should be configured on this Access Connector. Possible values are `SystemAssigned` and `UserAssigned`.

* `identity_ids` - (Optional) Specifies a list of User Assigned Managed Identity IDs to be assigned to this Access Connector.

~> **NOTE:** This is required when `type` is set to `UserAssigned`.
```

Reply:
Fixed.

Reply:
@WodansSon haha. Thanks man. In a rush to merge this one? :) I'd get to it tomorrow otherwise.

Reply:
Yes this is an APEX ask resource so there is release by date assigned to it... I would have normally asked, but time is of the essence at the moment... sorry 🙂